### PR TITLE
Search page breaks when posts exist without memberships

### PR DESCRIPTION
**Pull request by patcon**
Seems like some ~~roles~~ posts might be vacant (ie. a vice chair or a deceased member for a short while), and this should be accommodated.

### CODE REVIEW

**Comment by patcon** [comment]
Ready for review :)

File: councilmatic_core/models.py [hunk 1]
```diff
@@ -531,7 +531,7 @@ def organization(self):
 
     @property
     def current_member(self):
-        if self.memberships:
```
Comment (fgregg) [inline councilmatic_core/models.py]: Why is this necessary?

Comment (patcon) [inline councilmatic_core/models.py]: The membership object returns true even when there are no members, and so "self.memberships.first()` returns `None`, and the call for `membership.end_date` errors out

Comment (patcon) [inline councilmatic_core/models.py]: My impression from my earlier attempt to make councilmatic work without data was that this might have been a common oversight

**Comment by cathydeng** [comment]
nice catch, thanks :+1:
